packages/components/Molecules/Search/SearchResultsV2.vue

```diff
@@ -112,7 +112,7 @@ export default {
   @import "~@dpc-sdp/ripple-global/scss/settings";
   @import "~@dpc-sdp/ripple-global/scss/tools";
 
-  .rpl-search-results-layout {
+  .rpl-search-results {
```

Comment:
As css here is not scoped(I think same as original searchResults component), same class name here may effect on each others component? For example, user visit old search page and then v2 search page, both css will be loaded and old style will be applied on v2 page? May worth to double check and I still prefer to use different class name here in case it happened.

Reply:
Ah yes good point, I might go back to calling it searchresultslayout then I was hoping it to be a drop in replacement